recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt

```diff
@@ -214,6 +218,9 @@ class RecyclerViewFastScroller @JvmOverloads constructor(
 
             trackView.background = attribs.getDrawable(R.styleable.RecyclerViewFastScroller_trackDrawable)
 
+            if (it.getBoolean(R.styleable.RecyclerViewFastScroller_supportSwipeToRefresh,false))
```

Comment:
it's a good practice to have curly braces surrounding an `if` condition. Could you refactor this to change it?

Reply:
ok, I will do that